ansible/roles/vm_set/tasks/external_port.yml

```diff
@@ -4,7 +4,10 @@
   become: yes
   register: external_port_cfg
 
+- name: Set temp var to the {{ external_port }} configure file
+  set_fact: conf_file="/etc/network/interfaces.d/external_port"
```

Comment:
@vincent201881 it would be much better to move this definition to the top and reuse in template processing.

IMHO, the "temp" specifier is not valid here:
Set interface {{ external_port }} configuration file